tests/integration/base_test.py

```diff
@@ -42,24 +56,30 @@ def popup_check(self, mentoring, item_feedbacks, do_submit=True):
 
         submit = mentoring.find_element_by_css_selector('.submit input.input-main')
 
+        scenario_title = mentoring.find_element_by_css_selector('h2.main')
+
+        question_title = mentoring.find_element_by_css_selector('h3.question-title')
+
         for index, expected_feedback in enumerate(item_feedbacks):
             choice_wrapper = choices_list.find_elements_by_css_selector(".choice")[index]
             if do_submit:
                 # click on actual radio button:
                 choice_wrapper.find_element_by_css_selector(".choice-selector input").click()
                 submit.click()
+                self.wait_until_exists('.choice-result.checkmark-incorrect,.choice-result.checkmark-correct')
             item_feedback_icon = choice_wrapper.find_element_by_css_selector(".choice-result")
-            choice_wrapper.click()
+            # choice_wrapper.click()
```

Comment:
@omarkhan @jbzdak  I believe I have added it because the test was flaky - I can't recall the exact motivation, but most likely it has something to do with the fact that selenium *sometimes* fails to click an input element if it's not visible; while "clicking" normal DOM elements scrolls the viewport to the element. So, if it works without this - I'm totally fine with this line removed.